### DISCUSSION TITLE
`Ref` type accessibility in SFC

### DISCUSSION THREAD

**Discussion by webfansplz** [nuxt/nuxt#18175] [Questions]
I want to use `Ref` to type assertions, But: 

![image](https://user-images.githubusercontent.com/22515951/141795858-5d992bcf-9c53-46d4-b0ab-51112e551bf1.png)

**Comment by crotanite** [comment]
You can import Ref from vue like so: `import { Ref } from 'vue'`

**Comment by webfansplz** [comment]
https://github.com/nuxt/framework/issues/1928#issuecomment-968974991
